google-cloud-bigquery/benchmark/benchmark.rb

```diff
@@ -0,0 +1,54 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "google/cloud/bigquery"
+require "json"
+
+if ARGV.length < 1
```

Comment:
We're still reading `ENV["BIGQUERY_PROJECT"]` on line 23 right? Should that change too?

Reply:
Oops, yes, that should be changed as well.